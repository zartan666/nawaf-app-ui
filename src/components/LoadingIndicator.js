import React from 'react';
import {ActivityIndicator, View} from 'react-native';
import colors from './../common/colors';

const LoadingIndicator = ({style}) => (
  <ActivityIndicator
    size="small"
    animating
    color={colors.accent}
    style={{paddingVertical: 20}}
  />
);

LoadingIndicator.propTypes = {
  style: View.propTypes.style,
};

export default LoadingIndicator;
