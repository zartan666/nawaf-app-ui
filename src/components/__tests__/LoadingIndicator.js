import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import LoadingIndicator from './../LoadingIndicator';

const emitter = {
  emit: jest.fn().mockReturnThis(),
};

test('renders null', () => {
  const tree = renderer.create(<LoadingIndicator />).toJSON();
  expect(tree).toMatchSnapshot();
});
