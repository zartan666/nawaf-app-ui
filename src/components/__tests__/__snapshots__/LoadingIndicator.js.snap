// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders null 1`] = `
<ActivityIndicator
  animating={true}
  color="#ff1144"
  hidesWhenStopped={true}
  size="small"
  style={
    Object {
      "paddingVertical": 20,
    }
  }
/>
`;
